src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -39,12 +39,13 @@
     /**
      * Loads one or several .env files.
      *
-     * @param ...string A list of files to load
+     * @param string   $path  A file to load
+     * @param string[] $paths A list of additionnal files to load
```

Comment:
@nicolas-grekas says the opposite https://github.com/symfony/symfony/pull/22364#pullrequestreview-31925210